packages/client/src/components/TestToolBox.js

```diff
@@ -60,23 +55,35 @@ const TestToolBox = ({ address }) => {
             <div className="mr-5">
               <label>
                 <strong>Balances</strong>
-                <a className="is-underlined ml-2" onClick={updateUserBalance}>
+                <button
+                  href="#"
```

Comment:
Do we need href on a button? Does this change its view to link?

Reply:
Good catch! Will remove